tools/wheel_resolver/third_party/python/BUILD

```diff
@@ -0,0 +1,56 @@
+package(
```

Comment:
Is there a reason why the `tools/wheel_resolver/third_party` directory can't use `third_party` at the top level?